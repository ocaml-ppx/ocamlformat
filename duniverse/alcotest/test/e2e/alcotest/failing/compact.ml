let () =
  let open Alcotest in
  let passing s = test_case s `Quick (fun () -> ()) in
  let failing s = test_case s `Quick (fun () -> failwith "Error") in
  run ~compact:true __FILE__
    [
      ("alpha", [ passing "1"; passing "2"; failing "3"; passing "4" ]);
      ("beta", [ passing "1" ]);
      ("gamma", [ passing "1" ]);
    ]
