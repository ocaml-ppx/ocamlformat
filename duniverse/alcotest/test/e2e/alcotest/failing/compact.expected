Testing `test/e2e/alcotest/failing/compact.ml'.
This run has ID `<uuid>'.

..F...

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]        alpha          2   3.                                          │
└──────────────────────────────────────────────────────────────────────────────┘
[failure] Error

Logs saved to `<build-context>/_build/_tests/<test-dir>/alpha.002.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 6 tests run.
